base/system/logonui/NT5design.c

```diff
@@ -30,7 +30,7 @@ NT5_DrawLogoffCaptionText(LPWSTR lpText,
     LogFont.lfCharSet = DEFAULT_CHARSET;
     LogFont.lfHeight = 22;
     LogFont.lfWeight = 109; // From WinXP disassembly
```

Comment:
That comment is cringe.

Reply:
Well, it was [already like that](https://github.com/reactos/reactos/blob/master/base/system/logonui/NT5design.c#L32) when I found the file. I can remove the comment, if required.

Reply:
Hehe no it was just me reacting to what I saw.